bundles/config/org.eclipse.smarthome.config.discovery.test/src/test/groovy/org/eclipse/smarthome/config/setup/test/discovery/DiscoveryServiceMockOfBridge.groovy

```diff
@@ -0,0 +1,22 @@
+package org.eclipse.smarthome.config.setup.test.discovery
```

Comment:
Oh noooo, pleeease, no NEW groovy tests!

Reply:
It's just a copy of an existing test modified to check the new behavior. As the whole infrastructure for the DiscoveryService tests is in Groovy I see no benefit in duplicating it to Java.